### DISCUSSION TITLE
E3V2-BLTouch-5x5-v4.2.2-v2.0.1.bin problem

### DISCUSSION THREAD

**Discussion by anceco2002** [Jyers/Marlin#1805] [General Discussion]
good
I have a little problem, I have the e3v2 jyers 5x5 bltouch firmware installed, well now I want to install the e3d-v6 hotend on the printer. And this hotend, the thermistor that it carries, I have to put it in mode 5 in the firmware, the one that comes with v2 is in mode 1. And at the same time, raise the maximum temperature to 300, which would be 285, which is recommended for v6. How can I do it in the jyers .bin file because otherwise I would need the configuration files " configuration_adv.h and configuration.h" which are inside the jyers bin file of the bltouch 5x5 version.
This is the firmware that I have installed:
E3V2-BLTouch-5x5-v4.2.2-v2.0.1.bin

**Comment by BlueWings172** [comment]
to change the max temperature, you need to change the corresponding code in the firmware then compile. These are the steps in [this post](https://github.com/Jyers/Marlin/discussions/1787#discussioncomment-1978481)

For the thermistor, did the manufacturer tell you what changes you need to make in configuration_adv.h ?
